elements/pl-external-grader-results/pl-external-grader-results.py

```diff
@@ -139,17 +139,20 @@ def render(element_html, data):
                             test["max_points"] = round_value(
                                 results_test.get("max_points")
                             )
-                            correct = test["max_points"] == test["points"]
 
-                            # Don't show points for test cases that are 0/0.
-                            # Note that we compare with a string instead of a
-                            # number because we converted it to a string with
-                            # the above call to `round_value()`.
-                            if correct and test["max_points"] == "0":
-                                test["show_points"] = False
+                            if test["max_points"] != test["points"]:
+                                test["results_color"] = "#F44336"
```

Comment:
Are these standard (named) colors? Would be nice to use the mapping in `colors.py` to avoid having the raw color hex value in these strings. Not a huge deal but just a thought.

Reply:
These were the colors originally used here (see removed line 151 below). I did a quick script to find the closest colors from that list, and didn't get that close:
* for incorrect: tomato (ff6347)
* for correct: mediumseagreen (3cb371)

@mwest1066 or @nwalters512, should I switch these colors for named colors?

Reply:
We haven't touched this code in the five years since I wrote it. I'm not opposed to changing it, but I see extraordinarily little value in doing so.